server/matchmaker/matchmaker_queue.py

```diff
@@ -288,8 +290,7 @@ def to_dict(self):
                 ndigits=2
             ),
             "num_players": self.num_players,
-            "boundary_80s": [search.boundary_80 for search in self._queue.keys()],
-            "boundary_75s": [search.boundary_75 for search in self._queue.keys()],
+            "boundaries": boundaries,
```

Comment:
We can't just remove the old keys as that would be a breaking change in the protocol. The best thing to do would be to leave them, add a `# DEPRECATED:` comment indicating that they will be removed eventually, and add the new `boundaries` key in addition to the existing ones.

Somehow I thought they were already marked as deprecated though.

Reply:
@Askaholic What do you think about sending a list (set) of means instead of boundaries? Then the client can decide for itself what range it wants to use

Reply:
That sounds like a good idea

Reply:
You're right we should probably send the party mean instead of each players rating as the partys are matched by their average resting only. If a 1k and a 2k form a party they need other 1500s in the queue to match.

We don't need the number of players in a chunk, only if a chunk is populated at all. We could do it like this but 100 is probably a bit too coarse. How big of a deal is it anyway when the message size varies?

Reply:
Does not removing the old keys to stay protocol compatible also imply that they should still send the same info?
Breaking changes were planned for v2 of the server, right?

Reply:
Message size varying is fine, I want to make sure however that we think about the different ways that we could be encoding the information that the client needs and sending it in the most byte efficient way. More bytes means more CPU load on the server and more potential to run into bandwidth issues. If we can pass the same information in a way that avoids the possibility of scaling infinitely then I say we should go with that.

The protocol versioning is more about the structure of messages and not so much about the content. If certain messages start sending empty lists that used to be non-empty, that should be fine as the client should be able to handle that case anyway. Some feature of the client might stop working by showing empty results or just not appearing anymore, but it doesn't cause the client to break (crash) due to serialization errors or trying to perform operations on an incorrect type.